src/ajax/xhr.js

```diff
@@ -19,7 +19,19 @@ jQuery.ajaxSettings.xhr = window.ActiveXObject !== undefined ?
 			// and http://www.w3.org/Protocols/rfc2616/rfc2616-sec9.html#sec9
 			// Although this check for six methods instead of eight
 			// since IE also does not support "trace" and "connect"
-			/^(get|post|head|put|delete|options)$/i.test( this.type ) &&
+			//
+			// Support: IE 10-11
+			// IE seems to error on cross-domain PATCH requests when ActiveX XHR
+			// is used. Use the following strategy:
+			// 1. If request is local, use the ActiveX XHR.
+			// 2. Otherwise, in IE9+ always use the native XHR.
+			// 3. Otherwise, in IE8 if request type is *not* included in the following
+			//    list, use the ActiveX XHR.
+			// 4. Otherwise, use the native XHR.
+			// jshint -W018
+			( !( document.documentMode < 9 ) ||
+			// jshint +W018
+				/^(get|post|head|put|delete|options)$/i.test( this.type ) ) &&
```

Comment:
> ```js
( document.documentMode < 9 && /^patch$/i.test( this.type ) || createStandardXHR() ) && createActiveXHR();
```

IE<9 doesn't support any non-RFC2616 method as indicated in the comment. I don't think testing for `PATCH` is enough.